hangupsbot/plugins/getadmins.py

```diff
@@ -0,0 +1,22 @@
+# Original Author: kilr00y@esthar.net
+
+import hangups
+
+import plugins
+
+from hangups.user import UserList
+
+def _initialise(bot):
+    plugins.register_user_command(["admins"])
+
+def _get_adminlist(bot):
+    return bot.get_config_option('admins')
+
+def admins(bot,event,*args):
+    admin_list=_get_adminlist(bot)
```

Comment:
_get_adminlist() really does not need to exist, remove the extra function, and make the config call within one line :)